library/core/registry/src/main/java/org/quiltmc/qsl/registry/api/event/RegistryEntryContext.java

```diff
@@ -0,0 +1,39 @@
+package org.quiltmc.qsl.registry.api.event;
+
+import net.minecraft.util.Identifier;
+import net.minecraft.util.registry.Registry;
+
+/**
+ * Represents information about a registry entry.
+ *
+ * @param <V> the entry type used by the relevant {@link Registry}
+ */
+public interface RegistryEntryContext<V> {
+	/**
+	 * Getter for the relevant registry.
+	 *
+	 * {@return the relevant registry for this entry}
```

Comment:
Just saw this, but if you use the inline return notation, the javadoc will generate:

```
Returns the relevant registry for this entry.
Getter for the relevant registry.

@return - the relevant registry for this entry
```

Or something similar, so either keep the getter stuff and make the `@return` a non-inline notation, or remove the short description in favor of the inline notation.